cost-analyzer/charts/thanos/templates/query-frontend-deployment.yaml

```diff
@@ -25,6 +25,9 @@ spec:
       app.kubernetes.io/instance: {{ .Release.Name }}
       app.kubernetes.io/component: query-frontend
 {{ with .Values.queryFrontend.deploymentMatchLabels }}{{ toYaml . | indent 6 }}{{ end }}
+{{ with .Values.store.deploymentStrategy }}
```

Comment:
All of these use the store's deployment strategy. Is that what you intended?

Reply:
No, I will push up the change to fix this today.

Reply:
Fixed with https://github.com/kubecost/cost-analyzer-helm-chart/pull/1332